watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -589,52 +594,51 @@ def initialize_system(m, bio_P=True, solver=None):
         tear_guesses2 = {
             "flow_vol": {0: 0.003},
             "conc_mass_comp": {
-                (0, "S_A"): 0.1,
+                (0, "S_A"): 0.10,
                 (0, "S_F"): 0.16,
-                (0, "S_I"): 0.05745,
-                (0, "S_N2"): 0.025,
-                (0, "S_NH4"): 0.04,
-                (0, "S_NO3"): 1e-9,
-                (0, "S_O2"): 0.0014,
-                (0, "S_PO4"): 0.026,
+                (0, "S_I"): 0.057,
+                (0, "S_N2"): 0.036,
+                (0, "S_NH4"): 0.03,
+                (0, "S_NO3"): 0.002,
+                (0, "S_O2"): 0.0013,
+                (0, "S_PO4"): 0.024,
                 (0, "S_K"): 0.38,
-                (0, "S_Mg"): 0.028,
-                (0, "S_IC"): 0.075,
-                (0, "X_AUT"): 1e-9,
-                (0, "X_H"): 22.0,
-                (0, "X_I"): 10.8,
-                (0, "X_PAO"): 11.8,
-                (0, "X_PHA"): 0.0072,
-                (0, "X_PP"): 3.17,
-                (0, "X_S"): 3.71,
+                (0, "S_Mg"): 0.027,
+                (0, "S_IC"): 0.072,
+                (0, "X_AUT"): 0.25,
+                (0, "X_H"): 23.0,
+                (0, "X_I"): 11.3,
+                (0, "X_PAO"): 10.8,
+                (0, "X_PHA"): 0.0058,
+                (0, "X_PP"): 2.9,
+                (0, "X_S"): 3.8,
```

Comment:
Wondering if you came up with these guesses in a formulaic way, in relation to the feed composition, or if you came up with these purely through trial and error. Preferably, the tear guesses would have some known relationship with the feed composition so that we wouldn't need to manually adjust these when changing the feed.

Reply:
The tear guesses are supposed to be initial value we guessed for the place we cut the loop, we don't know the exact values and it cannot be directly derive from the feed composition. I get these guessed values from multiple attempts of simulations, it is relevant stable with feed composition, though, if we change the feed, the flowsheet can still be solved with the tear_guess I set.